images/ceph/Makefile

```diff
@@ -19,8 +19,8 @@ include ../image.mk
 
 CEPH_IMAGE = $(BUILD_REGISTRY)/ceph-$(GOARCH)
 
-CEPH_VERSION = v14.2.1-20190430
-BASEIMAGE = ceph/ceph-$(GOARCH):$(CEPH_VERSION)
+CEPH_VERSION = latest-nautilus-devel
```

Comment:
Since this is a temporary fix only in the master branch, let's use the following workaround temporarily. This way the integration tests can run on the amd64 image. However, we **must** update this to use an official `ceph/ceph` image before we release 1.1.
```
ifeq ($(GOARCH),amd64)
CEPH_VERSION = latest-nautilus-devel
BASEIMAGE = ceph/daemon-base-$(GOARCH):$(CEPH_VERSION)
else
CEPH_VERSION = v14.2.1-20190430
BASEIMAGE = ceph/ceph-$(GOARCH):$(CEPH_VERSION)
endif
```

Reply:
> Since this is a temporary fix only in the master branch, let's use the following workaround temporarily. This way the integration tests can run on the amd64 image. However, we **must** update this to use an official `ceph/ceph` image before we release 1.1.
> 
> ```
> ifeq ($(GOARCH),amd64)
> CEPH_VERSION = latest-nautilus-devel
> BASEIMAGE = ceph/daemon-base-$(GOARCH):$(CEPH_VERSION)
> else
> CEPH_VERSION = v14.2.1-20190430
> BASEIMAGE = ceph/ceph-$(GOARCH):$(CEPH_VERSION)
> endif
> ```

Thanks @travisn for the workaround!, @poornimag can you please address in this PR ?